packages/vite/src/node/optimizer/index.ts

```diff
@@ -1076,6 +1083,20 @@ export function getDepHash(config: ResolvedConfig, ssr: boolean): string {
   return getHash(content)
 }
 
+function getPatchesLastModificationTime(dir: string): string | undefined {
+  const fullPath = path.join(dir, 'patches')
```

Comment:
I feel we shouldn't make this assumption, as this is customizable with `--patch-dir` option of patch-package. Maybe we could have a better way to detect it?